18 Normalizr/src/pages/student-detail/studentDetailContainer.tsx

```diff
@@ -20,7 +22,8 @@ const mapDispatchToProps = (dispatch) => {
     fireFieldValueChanged: (
       viewModel: any,
       fieldName: string,
-      value: any) => dispatch(studentFieldValueChangedStart(viewModel, fieldName, value)
+      value: any,
+      subProperty?: string) => dispatch(studentFieldValueChangedStart(viewModel, fieldName, value, subProperty)
```

Comment:
I don't like changing a nested object, but also this seems to break the standard signature from other components (where 4th argument was filter).